docs/content/integrations/chef-automate.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Chef Automate
+subtitle: Operational Visibility Dashboard
+tutorials:
+- https://github.com/chef/automate/tree/master/components/authz-service#authz-with-opa
+blogs:
+- https://blog.verygoodsecurity.com/posts/building-a-fine-grained-permission-system-in-a-distributed-environment/
+- https://choria.io/blog/post/2020/02/14/rego_policies_opa/
```

Comment:
Both of these links have little to do with the Chef application.... Do you know what happened here? 🤔

Reply:
not sure, but they're in main at the moment: https://github.com/open-policy-agent/opa/blob/main/docs/website/data/integrations.yaml#L1403

Reply:
I can drop them as part of this if you think that's best.

Reply:
Yup, thanks!

Reply:
46a90f0686138a37ced80e35d14e2c45b9bdea46 ✔️